docs/index.md

```diff
@@ -91,17 +93,175 @@ nml_sdk.url = os.environ['NML_SDK_URL']
+If a `None` is provided, NannyML will calculate a sensible default during the first evaluation run.
+
+```python
+import nannyml_cloud_sdk as nml_sdk
+import os
+import pandas as pd
+
+nml_sdk.url = os.environ['NML_SDK_URL']
+nml_sdk.api_token = os.environ['NML_SDK_API_TOKEN']
+
+# Load a NannyML binary classification dataset to use as example
+reference_data = pd.read_csv('https://github.com/NannyML/nannyml/raw/main/nannyml/datasets/data/synthetic_car_loan_reference.csv')
+analysis_data = pd.read_csv('https://github.com/NannyML/nannyml/raw/main/nannyml/datasets/data/synthetic_car_loan_analysis.csv')
+target_data = pd.read_csv('https://github.com/NannyML/nannyml/raw/main/nannyml/datasets/data/synthetic_car_loan_analysis_target.csv')
+
+# The evaluation data will be a combination of analysis and target datasets. We'll only use the first 1000 rows for now.
+evaluation_data = analysis_data.merge(target_data, on='id').head(1000)
+
+print(reference_data.head())
+
+# Inspect schema from dataset and apply overrides
+schema = nml_sdk.model_evaluation.Schema.from_df(
+    'BINARY_CLASSIFICATION',
+    reference_data,
+    target_column_name='repaid',
+)
+
+# Create model
+model = nml_sdk.model_evaluation.Model.create(
+    schema,
+    reference_data=reference_data,
+    evaluation_data=evaluation_data,
+    metrics_configuration={
+        'F1': {
+            'enabled': True,
+            'rope_lower_bound': 0.8,
+            'rope_upper_bound': 0.9,
+            'hdi_width': 0.01
+        },
+        'ACCURACY': {
+            'enabled': True,
+            'rope_lower_bound': None,
+            'rope_upper_bound': None,
+            'hdi_width': None
+        },
+    },
+    name='from_sdk',
+    key_performance_metric='F1',
+    hypothesis='MODEL_PERFORMANCE_NO_WORSE_THAN_REFERENCE',
+    classification_threshold=0.5,
+)
+print("Model", model['id'], "created at", model['createdAt'])
+
+# Now trigger the model evaluation run with our first 1000 rows of evaluation data
+
+nml_sdk.model_evaluation.Run.trigger(model['id'])
+```
+
+Now we'll add the next set of evaluation data and trigger another evaluation run.
+
+```
+import nannyml_cloud_sdk as nml_sdk
+import os
+import pandas as pd
+
+evaluation_data = analysis_data.join(target_data, on='id').iloc[1000:2000]
+
+# Retrieve the model that was created earlier, using the name as a filter.
+model, = nml_sdk.model_evaluation.Model.list(name='from_sdk')
+print(model)
+
+# Adding the new evaluation data to the model
+nml_sdk.model_evaluation.Model.add_evaluation_data(model_id=model['id'], data=evaluation_data)
+
+# Trigger the model evaluation run to include the latest set of evaluation data
+nml_sdk.model_evaluation.Run.trigger(model['id'])
+```
+
+### Experiments
+
+This snippet provides an example of how you can set up an A/B-testing experiment in NannyML Cloud.
+
+You can configure each of the available metrics by providing an optional value for ROPE and HDI width.
```

Comment:
The parameters are required for experiments since we don't have reference data available

Reply:
Woopsie, copy pasta